src/metabase/models/database.clj

```diff
@@ -343,3 +349,8 @@
 (defmethod serdes/storage-path "Database" [{:keys [name]} _]
   ;; ["databases" "db_name" "db_name"] directory for the database with same-named file inside.
   ["databases" name name])
+
+(defn json-unfolding-default
+  "Returns true if JSON fields should be unfolded by default for this database, and false otherwise."
```

Comment:
it'd be nice to give a bit of context why `nil` is treated as `true` here.

Reply:
Added on line 356-358